Release/5.4.1 - merge into release/5.5 branch

AmandaRiu opened the pull request:
Merge `release/5.4.1` branch into `release/5.5`. This branch includes the Hotfix for #3183 

Update release notes:

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

jkmassel committed:
Merge pull request #1444 from woocommerce/merge/2.7-into-master

Merge 2.7 into master

jkmassel committed:
Update Release notes for 2.8

jkmassel committed:
Update metadata strings

jkmassel committed:
Merge pull request #1454 from woocommerce/merge/2.8-into-master

Merge 2.8 Changes + Code Freeze into master

jkmassel committed:
Merge pull request #1485 from woocommerce/merge/2.8-into-master

Merge/2.8 into master

jkmassel committed:
Merge pull request #1487 from woocommerce/release/2.9

Merge 2.9 into master

JavonDavis committed:
Merge pull request #1500 from woocommerce/release/2.9

Merge release into master

JavonDavis committed:
Merge pull request #1528 from woocommerce/release/2.9

Merge 2.9 into master

JavonDavis committed:
Merge pull request #1536 from woocommerce/release/3.0

Merge 3.0 into master

JavonDavis committed:
Updates translations

JavonDavis committed:
Bump version number

JavonDavis committed:
Update metadata translations for 3.0

JavonDavis committed:
Merge pull request #1574 from woocommerce/release/3.1

Merge 3.1 into master

JavonDavis committed:
Merge branch 'develop' into release/3.0

JavonDavis committed:
Merge pull request #1569 from woocommerce/release/3.0

Merge 3.0 into master

jkmassel committed:
Merge pull request #1593 from woocommerce/release/3.1

Merge 3.1 into master

JavonDavis committed:
Bump version number

jkmassel committed:
Updates translations

jkmassel committed:
Bump version number

jkmassel committed:
Update metadata translations for 3.1

Commit:
Cap logfile size to 63k characters for the Zendesk system status report

jkmassel committed:
Merge pull request #1620 from woocommerce/merge/3.1-into-master

Merge 3.1 into master

jkmassel committed:
Merge branch 'master' into merge/3.2-into-master

jkmassel committed:
Merge pull request #1627 from woocommerce/merge/3.2-into-master

Merge 3.2 into master

jkmassel committed:
Merge pull request #1645 from woocommerce/merge/3.1.1-into-master

Merge 3.1.1 hotfix into `master`

jkmassel committed:
Merge pull request #1649 from woocommerce/merge/release-notes-into-master

Merge release notes (and new beta) into master

AmandaRiu committed:
Merge pull request #1712 from woocommerce/merge/3.2-into-master

Merge 3.2 into master

jkmassel committed:
Merge pull request #1740 from woocommerce/release/3.3

Merge Release/3.3 into master

Commit:
Update metadata translations for 3.3

AmandaRiu committed:
Merge pull request #1797 from woocommerce/release/merge-3.3-master

Release/merge 3.3 into master

AmandaRiu committed:
Merge pull request #1818 from woocommerce/merge/3.4-into-master

Merge 3.4 code freeze into master

AmandaRiu committed:
Merge pull request #1832 from woocommerce/merge/3.4-release-notes-into-master

Merge 3.4 release notes and rc-2 into master

jkmassel committed:
Merge pull request #1851 from woocommerce/merge/3.4-final-into-master

Merge 3.4 final into master

jkmassel committed:
Merge pull request #1858 from woocommerce/merge/3.5-code-freeze-into-master

Merge 3.5 code freeze into master

jkmassel committed:
Add 3.5 final release notes

jkmassel committed:
Update metadata strings

nbradbury committed:
Merge pull request #1874 from woocommerce/merge/3.5-release-notes-into-master

Merge 3.5 release notes into master

jkmassel committed:
Merge pull request #1920 from woocommerce/merge/3.5-final-into-master

Merge 3.5 final into master

oguzkocer committed:
Merge pull request #1925 from woocommerce/merge/3.6-code-freeze-into-master

Merge 3.6 code freeze into master

oguzkocer committed:
Merge pull request #1943 from woocommerce/merge/3.6-release-notes-into-master

Merge 3.6 release notes into master

oguzkocer committed:
Merge pull request #1985 from woocommerce/merge/3.6-final-into-master

Merge 3.6 final into master

oguzkocer committed:
Merge pull request #1991 from woocommerce/merge/3.7-code-freeze-into-master

Merge 3.7 code freeze into master

oguzkocer committed:
Merge pull request #2013 from woocommerce/merge/3.7-release-notes-into-master

Merge 3.7 release notes into master

oguzkocer committed:
Merge pull request #2030 from woocommerce/merge/3.7-final-into-master

Merge 3.7 beta2, version info, metadata into master

oguzkocer committed:
Merge pull request #2039 from woocommerce/merge/3.8-code-freeze-into-master

Merge 3.8 code freeze into master

oguzkocer committed:
Merge pull request #2060 from woocommerce/merge/3.8-release-notes-into-master

Merge 3.8 release notes into master

oguzkocer committed:
Merge pull request #2106 from woocommerce/merge/3.8-final-into-master

Merge 3.8 final into master

oguzkocer committed:
Merge pull request #2109 from woocommerce/merge/3.9-code-freeze-into-master

Merge 3.9 code freeze into master

jkmassel committed:
Merge pull request #2142 from woocommerce/merge/3.9-release-notes-into-master

Merge 3.9 release notes into master

loremattei committed:
Merge pull request #2196 from woocommerce/merge/3.9-final-into-master

Merge 3.9 final into master

oguzkocer committed:
Merge pull request #2203 from woocommerce/merge/4.0-code-freeze-to-master

Merge 4.0 code freeze to master

loremattei committed:
Merge pull request #2222 from woocommerce/merge/4.0-release-notes-to-master

Merge 4.0 release notes to master

oguzkocer committed:
Merge pull request #2274 from woocommerce/merge/4.0-final-into-master

Merge 4.0 final into master

loremattei committed:
Merge pull request #2309 from woocommerce/merge/4.1-release-notes-to-master

Merge 4.1 to master

loremattei committed:
Merge pull request #2359 from woocommerce/merge/4.1-final-to-master

Merge 4.1 final to master

loremattei committed:
Merge pull request #2367 from woocommerce/merge/4.2-rc-1-to-master

Merge 4.2-rc-1 to master

loremattei committed:
Merge pull request #2381 from woocommerce/merge/4.2-release-notes-to-master

Merge 4.2 release notes to master

loremattei committed:
Merge pull request #2438 from woocommerce/merge/4.2-final-to-master

Merge 4.2 final to master

loremattei committed:
Merge pull request #2447 from woocommerce/merge/4.3-code-freeze-to-master

Merge 4.3 code freeze to master

loremattei committed:
Merge pull request #2475 from woocommerce/merge/4.3-release-notes-to-master

Merge 4.3 release notes and 4.2.1 hotfix to master

loremattei committed:
Merge pull request #2505 from woocommerce/merge/4.3-final-to-master

Merge 4.3 final to master

oguzkocer committed:
Merge pull request #2518 from woocommerce/release/4.4

Merge Release/4.4 back into master

oguzkocer committed:
Merge pull request #2535 from woocommerce/release/4.4

Release/4.4

oguzkocer committed:
Merge pull request #2552 from woocommerce/release/4.5

Merge Release/4.5 into Master

loremattei committed:
Merge pull request #2576 from woocommerce/release/4.5

Release/4.5

loremattei committed:
Merge pull request #2586 from woocommerce/release/4.6

Merge 4.6-rc-1 into Master

oguzkocer committed:
Merge pull request #2607 from woocommerce/release/4.6

Merge 4.6 release notes into Master

loremattei committed:
Merge pull request #2619 from woocommerce/release/4.6

Merge Release/4.6 into Master

jkmassel committed:
Merge pull request #2653 from woocommerce/release/4.7

Merge 4.7 code freeze and editorial to master

oguzkocer committed:
Merge pull request #2704 from woocommerce/release/4.7

Release/4.7

loremattei committed:
Merge pull request #2716 from woocommerce/release/4.8

Merge Release/4.8 branch into master

oguzkocer committed:
Merge pull request #2762 from woocommerce/release/4.8

Release/4.8

oguzkocer committed:
Merge pull request #2826 from woocommerce/release/4.9

Release/4.9

jkmassel committed:
Merge pull request #2865 from woocommerce/release/5.0

Release/5.0

loremattei committed:
Merge pull request #2951 from woocommerce/release/5.1

Release/5.1

loremattei committed:
Merge pull request #3037 from woocommerce/release/5.2

Release/5.2

loremattei committed:
Merge pull request #3115 from woocommerce/release/5.3

Merge Release/5.3 back into trunk

mokagio committed:
Merge release 5.4 (#3163)

Commit:
Update FluxC commit hash to pull in the hotfix for empty order product variation keys

Commit:
Update release notes for hotfix 5.4.1

0nko committed:
Merge pull request #3188 from woocommerce/hotfix/3183-crash-fix

Update FluxC commit hash to pull in the hotfix for empty order product variation keys

AliSoftware committed:
Update version in build.gradle (5.4.1 / 171)

AliSoftware commented on fastlane/metadata/android/de-DE/changelogs/.txt:
```diff
@@ -0,0 +1,2 @@
+3.1:
+- Du kannst dich nicht mehr an deine Anmeldedaten von WordPress.com erinnern? Kein Problem! Wir haben eine neue Option hinzugefügt, mit der Benutzer sich mit ihren Website-Anmeldedaten anmelden können. Wenn diese Anmeldedaten funktionieren, ruft die App die Informationen des verknüpften WordPress.com-Kontos ab und ermöglicht dem Benutzer, die Anmeldung über einen magischen Link abzuschließen.
```
Any idea where those come from? Seems those files were created in …2019 (e.g. this one is from a commit by @jkmassel)? No idea why they appear in this PR then?! 🤔

AliSoftware committed:
Cleanup old changelog files

Those files dates from 3.0 and 3.1, probably a leftover of a hotfix at that time introducing them in trunk but never back-merging them in develop, hence them popping back up on this hotfix cut from trunk

AliSoftware commented:
Superseded by https://github.com/woocommerce/woocommerce-android/pull/3193 (to fix the merge conflict)
